aws/scripts/py/deploy_content.py

```
@@ -50,13 +51,15 @@
 
 def deploy_files(bucket_name):
     print(f"Deploying files to bucket: {bucket_name}")
-    return subprocess.check_output(['aws', 's3', 'sync', './out', f's3://{bucket_name}'])
+    return subprocess.check_output(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)